plugins/zify/zify.mli

```diff
@@ -0,0 +1,29 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *   INRIA, CNRS and contributors - Copyright 1999-2019       *)
+(* <O___,, *       (see CREDITS file for the list of authors)           *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
+open Constrexpr
+module InjTable : sig val register : constr_expr -> unit end
```

Comment:
I think it may be nicer if you had
~~~ocaml
module type Table = sig 
  val register : constr_expr -> unit 
  val print : unit -> unit 
end
module InjTable : Table
module UnOp : Table 
(* ... *)
~~~
(with the print functions kept inside the module instead of listed at the end of the mli)

Reply:
done.